src/index/Vocabulary.cpp

```diff
@@ -187,7 +203,8 @@ std::optional<IdRange<I>> Vocabulary<S, C, I>::getIdRangeForFullTextPrefix(
     const string& word) const {
   AD_CONTRACT_CHECK(word[word.size() - 1] == PREFIX_CHAR);
   IdRange<I> range;
-  auto prefixRange = prefix_range(word.substr(0, word.size() - 1));
+  auto prefixRange =
+      prefix_range_internal_vocabulary(word.substr(0, word.size() - 1));
```

Comment:
Actually, how big/small is the text index vocabulary, would there be benefit in also making it externalizable?